src/app/core/services/filters.service.ts

```diff
@@ -4,17 +4,30 @@ import { BehaviorSubject, pipe } from 'rxjs';
 
 export type Filter = {
   title: string;
-  status: string;
+  status: string[];
   type: string;
   sort: Sort;
   labels: string[];
   milestones: string[];
   hiddenLabels?: Set<string>;
 };
 
+type StatusInfo = {
+  type: string;
+  status: string;
+};
+
+/**
+ * Converts a status string into an info object
+ */
+export const statusToType = (statusString: string): StatusInfo => {
+  const [status, type] = statusString.split(' ');
+  return { status, type };
+};
```

Comment:
Is it necessary to include a statusToType sort of function in the filters service? 
The filters service should be exclusively kept for storing and updating filters in line with the SRP.
I think this characterization of information from certain filters should be done where the filters are applied i.e., in dropdownfilter.ts in this case.
I would suggest simply using the status string in dropdownfilter.ts and parsing it appropriately as part of filtering

Reply:
Fixed!